frontend/src/components/Collections/components/Dataset/components/DownloadDataset/components/Content/components/DataFormat/constants.ts

```diff
@@ -1,3 +1,33 @@
+/**
+ * @deprecated by TOOLTIP_SLOT_PROPS constant once "DOWNLOAD_UX" feature flag is removed (#5566).
+ */
+export const DOWNLOAD_TOOLTIP_SLOT_PROPS = {
```

Comment:
Rather than using `blueprintjs` tooltip, I've styled `czi-sds` tooltip to mimic the `blueprintjs` tooltip.